process-services/1.11/config/index.md

```diff
@@ -0,0 +1,180 @@
+
+    If you then run using `catalina.sh` you will see the secret password in the logging on application startup. This is a Tomcat feature, which you can disable by removing `<Listener className="org.apache.catalina.startup.VersionLoggerListener" />` from your Tomcat's `server.xml` [https://stackoverflow.com/questions/35485826/turn-off-tomcat-logging-via-spring-boot-application](https://stackoverflow.com/questions/35485826/turn-off-tomcat-logging-via-spring-boot-application){:target="_blank"} You may initially, however, want to leave this on for diagnostic purposes until you’ve proven you’ve got encryption working. For an example of this, see [https://stackoverflow.com/questions/17019233/pass-user-defined-environment-variable-to-tomcat](https://stackoverflow.com/questions/17019233/pass-user-defined-environment-variable-to-tomcat){:target="_blank"}.
+
+    For other servers there will be other ways of setting environment/JVM variables. These values can be read as JVM parameters, environment variables or as property file entries (though you would not want to put the secret encryption password in a property file). Therefore, with WebSphere they could set using JVM parameter config [http://www-01.ibm.com/support/docview.wss?uid=swg21417365](http://www-01.ibm.com/support/docview.wss?uid=swg21417365){:target="_blank"} or environment variable config [https://www.ibm.com/support/knowledgecenter/en/SSAW57_8.5.5/com.ibm.websphere.nd.doc/ae/welcvariables.html.](https://www.ibm.com/support/knowledgecenter/en/SSAW57_8.5.5/com.ibm.websphere.nd.doc/ae/welcvariables.html){:target="_blank"}.
+
+8. Start the application.
+
+    The application should now start as normal. If it doesn’t, try without the encrypted values and without the encryption parameters to determine whether the problem is related to the encryption setup. Check that you are able to encrypt and decrypt with Jasypt to rule out any issues due to copy-paste errors. 
+
+9. Logging.
+
+    Some property values (though not sensitive ones) are logged by Alfresco applications if the log level is set high. If you want to restrict this then reduce the log level in`log4j.properties`
+
+## CORS
+
+To enable Cross Origin Resource Sharing (CORS) in Process Services, set the `cors.enabled` property to true in the
+`activiti-app.properties` file.
+
+>**Note:** This feature is only supported on Tomcat application server.
+
+```text
+# CORS CONFIGURATION
+#
+cors.enabled=true
+```
+
+When CORS is enabled, CORS requests can be made to all endpoints under `/activiti-app/api`.
+
+Also, some additional properties are made available which can be configured to further fine tune CORS. This will make CORS available only to certain origins or to restrict the valid HTTP methods that can be used and headers that can be sent with CORS-enabled requests.
+
+```text
+cors.enabled=false
+cors.allowed.origins=*
```

Comment:
`*` can no longer used as the value for this property. A new property `cors.allowed.origin.patterns` should be used in case `*` needs to be used.

Reply:
Same as other one. only property name changes remains =*

Reply:
Updated